crate/operator/utils/kubeapi.py

```diff
@@ -236,3 +236,16 @@ async def get_cratedb_resource(namespace: str, name: str) -> dict:
             namespace=namespace,
             name=name,
         )
+
+
+async def get_gc_user_password(core: CoreV1Api, namespace: str, name: str) -> str:
+    """
+    Return the password for the grand-central user of cluster ``name`` in ``namespace``.
+
+    :param core: An instance of the Kubernetes Core V1 API.
+    :param namespace: The namespace where the CrateDB cluster is deployed.
+    :param name: The name of the CrateDB cluster.
+    """
+    return await resolve_secret_key_ref(
+        core, namespace, {"key": "password", "name": f"user-gc-{name}"}
```

Comment:
`f"user-gc-{name}"` is hard-coded in many places, maybe you could add a constant for it